.github/workflows/release.yml

```diff
@@ -32,36 +24,26 @@ jobs:
           force: ${{ github.event.inputs.force }}
       - uses: actions/checkout@v2
         with:
-          token: ${{ secrets.GH_SENTRY_BOT_PAT }}
-          fetch-depth: 20
+          token: ${{ secrets.GH_RELEASE_PAT }}
       - uses: getsentry/craft@master
         name: Craft Prepare
-        if: ${{ !github.event.inputs.skip_prepare }}
         with:
           action: prepare
           version: ${{ env.RELEASE_VERSION }}
         env:
           ZEUS_API_TOKEN: ${{ secrets.ZEUS_API_TOKEN }}
-      # Wait until the builds start. Craft should do this automatically
-      # but it is broken now.
-      - run: sleep 10
-      - uses: getsentry/craft@master
-        name: Craft Publish
+      - name: Request publish
+        if: success()
+        uses: actions/github-script@v3
         with:
-          action: publish
-          version: ${{ env.RELEASE_VERSION }}
-        env:
-          DRY_RUN: ${{ github.event.inputs.dry_run }}
-          GITHUB_API_TOKEN: ${{ secrets.GH_SENTRY_BOT_PAT }}
-          ZEUS_API_TOKEN: ${{ secrets.ZEUS_API_TOKEN }}
-          TWINE_USERNAME: '__token__'
-          TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
-          DOCKER_USERNAME: 'sentrybuilder'
-          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
-      - id: next-dev-version
-        name: Set new version for development
-        if: ${{ !github.event.inputs.dry_run }}
-        run: |
-          git checkout master
-          ./scripts/bump-version.sh '' $(date -d "$(echo $RELEASE_VERSION | sed -e 's/^\([0-9]\{2\}\)\.\([0-9]\{1,2\}\)\.[0-9]\+$/20\1-\2-1/') 1 month" +%y.%-m.0.dev0)
-          git diff --quiet || git commit -anm 'meta: Bump new development version' && git pull --rebase && git push
+          # TODO: Replace token with GH_RELEASE_PAT after verification
+          token: ${{ secrets.GH_SENTRY_BOT_PAT }}
+          script: |
+            const repoInfo = context.repo;
+            await github.issues.createComment({
+              owner: repoInfo.owner,
+              repo: 'publish',
+              title: `publish ${repoInfo.repo}@{process.env.RELEASE_VERSION}`,
+              // TODO: Remove the following line after verification
+              labels: ['dry-run'],
```

Comment:
👀

Reply:
The thing is, to be able to add labels to issues, you need push permissions to the repo. Hence the comment. After all, maybe I should have gone with your "no-dry-run" label 😛

Reply:
Yup, I wuz just noticing what we talked about earlier. ☺️